README.md

```diff
@@ -34,15 +34,41 @@ pipx ensurepath
 
 Upgrade pipx with `brew update && brew upgrade pipx`.
 
-Otherwise, install via pip (requires pip 19.0 or later):
+### On Linux, install via pip (requires pip 19.0 or later)
 
 ```
 python3 -m pip install --user pipx
 python3 -m pipx ensurepath
 ```
+Upgrade pipx with `python3 -m pip install --user -U pipx`.
+
+### On Windows, install via pip (requires pip 19.0 or later)
+
+```
+# If you installed python using the app-store, replace `python` with `python3` in the next line.
```

Comment:
Hi. I'm using the store version. I don't get the reason why `python3` matters here.

Reply:
> Hi. I'm using the store version. I don't get the reason why `python3` matters here.

I guess at the time it was the thing to do for windows, still making a distinction between 2.7 and python3. I wonder if using `python3` still works or whether we need to change the documentation ? (I am not running windows anymore so unable to check)

Reply:
python and python3 is the same thing on win.

Reply:
Alright. So I guess we should change this

Reply:
Besides, if you say *pipx ensurepath will add both the above mentioned path and the `%USERPROFILE%\.local\bin` folder to your search path*, why `python3 -m pipx ensurepath` wouldn't?

Honestly I prefer the previous guide. I don't see a difference between Linux and Windows installation. The `Otherwise` is enough.

Though there are indeed some differences between store and nornal version. For example you don't need `--user` when using store version.

Reply:
The previous guide did not work. I can confirm this as I was on windows at that time. It could be that there is a difference in using python from the store or an install downloaded from pypi.org (which I did and also might have changed since this doc was written). In the end, the goal is getting your search paths right. So maybe there is not one solution that works in all windows situations.

Reply:
Then I bet that you met https://github.com/ofek/userpath/issues/25

If so, I would suggest reverting this PR.

Reply:
Basically the new instruction tells you what to do when something goes wrong. Which, in my situation (and other's) was the case (and many times over and over again)
I don't see removing that does help improve the instruction. 

If you can confirm the old instructions worked for a store based installation, maybe that could be described as the happy/default flow. And then add an "if things go wrong" section describing a possible solution.

But as I am no longer on windows anymore I am no longer in the right position to do this and make the change. So maybe add another pr making the change ?

Reply:
I can confirm at my that time it didn't work for store version.

The point is that I don't believe *go to the mentioned folder, run the pipx executable directly* makes difference with `python -m pipx ensurepath`.
The reaon you succeed is because you said you changed store version to normal one. You can see my test at `userpath#25`.

I'm still using store version. However, I just tested and find `python -m pipx ensurepath` now works, so do `userpath append`. So I can't check whether *run pipx directly* makes difference, either.